src/control.cc

```diff
@@ -339,13 +339,55 @@ AbstractNode *ControlModule::instantiate(const Context* ctx, const ModuleInstant
 
 void register_builtin_control()
 {
-	Builtins::init("child", new ControlModule(ControlModule::Type::CHILD));
-	Builtins::init("children", new ControlModule(ControlModule::Type::CHILDREN));
-	Builtins::init("echo", new ControlModule(ControlModule::Type::ECHO));
-	Builtins::init("assert", new ControlModule(ControlModule::Type::ASSERT));
 	Builtins::init("assign", new ControlModule(ControlModule::Type::ASSIGN));
-	Builtins::init("for", new ControlModule(ControlModule::Type::FOR));
-	Builtins::init("let", new ControlModule(ControlModule::Type::LET));
-	Builtins::init("intersection_for", new ControlModule(ControlModule::Type::INT_FOR));
-	Builtins::init("if", new ControlModule(ControlModule::Type::IF));
+
+	Builtins::init("child", new ControlModule(ControlModule::Type::CHILD),
```

Comment:
As `child()` is deprecated, we probably don't want to show calltips same as with the `import_*` modules.